emoji/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion versions.targetSdk
+        versionCode 1
+        versionName "0.1"
```

Comment:
This is following the lib version? If not this is not following the sem. versioning (that we use in the project as well).

Reply:
I'll change to sem. versioning :+1:

Reply:
Since this is an internal module, we really don't need any custom versioning here, "0.1" is just fine.